src/metabase/driver/h2.clj

```diff
@@ -218,8 +218,8 @@
   (merge (sql/IDriverSQLDefaultsMixin)
          {:date-interval                     (u/drop-first-arg date-interval)
           :details-fields                    (constantly [{:name         "db"
-                                                           :display-name "Connection String"
-                                                           :placeholder  "file:/Users/camsaul/bird_sightings/toucans"
+                                                           :display-name (tru "Connection String")
+                                                           :placeholder  (tru "file:/Users/camsaul/bird_sightings/toucans")
```

Comment:
Don't think we should i18n `file:/`, altho it would be funny for us, it wouldn't be so funny when someone tries to use `archivo:/` and can't figure out why it won't work. So maybe we should do `(str "file:" (trs ...))` or something

Reply:
I assumed we wouldn't translate the `file:/` part, but if I split it up like you suggest, we'll be sure that won't happen.